core/src/main/res/values/arrays.xml

```diff
@@ -269,6 +269,7 @@
         <item>is_favorite</item>
     </string-array>
 
+
```

Comment:
I think the two arrays above are unused now. Please remove them.

Reply:
got it